### PR TITLE
Move ada_launch functionality here

### PR DESCRIPTION
**Pull request by egordon**
There is no reason to separate ADA-specific launch files from the main library.

Note that the actual launch routines are usable, but a little disorganized (filed into https://github.com/personalrobotics/libada/issues/55).

Also: updates package.xml to reflect current maintainers.

***

**Before creating a pull request**

- [ N/A ] Document new methods and classes
- [ N/A ] Format code with `make format`

**Before merging a pull request**

- [ N/A ] Add unit test(s) for this change

### CODE REVIEW

File: launch/default.launch [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<launch>
+
+  <arg name="feeding" default="false" doc="Boolean flag of whether to set up the robot for the feeding demo"/>
+  <arg name="perception" default="true" doc="Boolean flag of whether to set up the real perception for the feeding demo"/>
+  <arg name="detector" default="retinanet" doc="Which network to use for food perception."/>
```
Comment (amalnanavati) [inline launch/default.launch:5]: This is a small thing, but could you write the alternatives to retinanet in a connect above this arg?

File: launch/default.launch [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          type="jaco_hardware"
+          output="screen" />
+
+   <!-- load controllers -->
+   <rosparam file="$(find ada_launch)/config/controllers.yaml" command="load" />
+
+   <node name="controller_spawner_started" pkg="controller_manager" type="spawner" respawn="false"
+      output="screen"
+      args="
+         joint_state_controller
```
Comment (amalnanavati) [inline launch/default.launch:19]: Is there a reason this is spaced out this way?

File: scripts/start_ada_tmux.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Start ADA functionalities in tmux sessions
```
Comment (amalnanavati) [inline scripts/start_ada_tmux.sh:2]: Could you add a comment here, or instructions in a readme, specifying what arguments to call this script with? (e.g., a sample terminal command to run it).

**Review by amalnanavati — COMMENTED** [review]
LGTM, I feel like it'll be ready to merge after addressing/discussing the above line-by-line comments.

File: launch/default.launch (hunk 2 above)
Comment (egordon) [reply launch/default.launch]: In order to give each controller name it's own line. So it is easy to see on git review which controllers were added / removed.

However, the "started" controllers are unlikely to contain anything but the joint_state_controller for the foreseeable future, so I'll make this more compact.

File: scripts/start_ada_tmux.sh (hunk 3 above)
Comment (egordon) [reply scripts/start_ada_tmux.sh]: Whoa, this is a very old file that no longer works (e.g. it references "ros_control.launch", which hasn't existed in the repo for years).

I'll just remove it.

**Comment by egordon** [comment]
@amalnanavati Thanks for the review! Should be ready for re-review

**Review by amalnanavati — APPROVED** [review]
LGTM! :)

**Pull request merged**